src/admin/views.py

```diff
@@ -15,3 +17,6 @@ def get(self, request):
             issues += challenge.self_check()
 
         return FormattedResponse(issues)
+
+def mail_list(request):
+    return render(request, "mail_list.html", context={"emails": getattr(mail, 'outbox', [])})
```

Comment:
Why do we need a new view for this? The console backend is good enough for checking whether emails would have been sent or not.

Reply:
I felt that a web view would make troubleshooting easier, particularly for things like SMTP email